### PR TITLE
Add support for Ivy

### PR DESCRIPTION
**Pull request by marjan-georgiev**
- Updates dependencies to make them compatible with Ivy
- Enables the `enableIvy` flag on the demo page
- Resolves Ivy compilation issues

### CODE REVIEW

File: projects/swimlane/ngx-ui/package.json [hunk 1]
```diff
@@ -45,9 +45,9 @@
     "json-schema": "*"
   },
   "dependencies": {
-    "@angular/flex-layout": "8.0.0-beta.26",
+    "@angular/flex-layout": "8.0.0-beta.27",
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/package.json]: Are we sure about this? I think it's safe to double check that the splitDirective is still working.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -5,10 +5,10 @@ import { FormsModule } from '@angular/forms';
 import { NgxDatatableModule } from '@swimlane/ngx-datatable';
 
 import { MomentModule } from 'ngx-moment';
-import { NgxUIModule } from '../../projects/swimlane/ngx-ui/src/public_api';
+import { NgxUIModule } from '@swimlane/ngx-ui';
```
Comment (duredhel13) [inline src/app/app.module.ts]: Wouldn't this make it a bit harder to update the docs?

Comment (marjan-georgiev) [reply src/app/app.module.ts]: It's still importing directly from source, I updated the paths. See here: https://github.com/swimlane/ngx-ui/pull/304/files#diff-e5e546dd2eb0351f813d63d1b39dbc48L17-L18

File: projects/swimlane/ngx-ui/package.json (hunk 1 above)
Comment (marjan-georgiev) [reply projects/swimlane/ngx-ui/package.json]: Good catch. Updated the demo pages. Looks like it doesn't like using the directive name as an input:

`<div [ngxSplit]="'row'">`

but likes this:

`<div ngxSplit [ngxSplit]="row">`

Comment (marjan-georgiev) [reply projects/swimlane/ngx-ui/package.json]: But that's probably Ivy-related, not flex-layout necessarily.

**Pull request merged**